foodalert/views.py

```diff
@@ -331,6 +333,7 @@ def get_context_data(self, *args, **kwargs):
         context['logout_url'] = settings.LOGOUT_URL
         context['ga_key'] = settings.GOOGLE_ANALYTICS_KEY
         context['debug_mode'] = settings.DEBUG
+        context['twilio_number'] = settings.TWILIO_FROM_NUMBER
```

Comment:
We should really stop calling settings attributes directly, but use getattr() and set a sensible default where possible or raise ImpoperlyConfigured. Doesn't need to for this merge, but we should get a TODO task created.